LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/Region.cs

```diff
@@ -133,14 +133,6 @@ public virtual bool TryGetJoinChannelIndex(double frequency, out int channelInde
         [Obsolete("#655 - This Rxpk based implementation will go away as soon as the complete LNS implementation is done.")]
         public abstract bool TryGetDownstreamChannelFrequency(Rxpk upstreamChannel, out double frequency, DeviceJoinInfo deviceJoinInfo = null);
 
-        /// <summary>
-        /// Implements logic to get the correct downstream transmission frequency for the given region based on the upstream channel frequency.
-        /// </summary>
-        /// <param name="upstreamFrequency">Frequency of the upstream message.</param>
-        /// <param name="dataRate">Ustream data rate.</param>
-        /// <param name="deviceJoinInfo">Join info for the device, if applicable.</param>
-        public abstract bool TryGetDownstreamChannelFrequency(double upstreamFrequency, ushort dataRate, out double downstreamFrequency, DeviceJoinInfo deviceJoinInfo = null);
```

Comment:
Additional cleanup, thanks 🙌

Reply:
Yeah, I didn't want to migrate to the DI logger for dead code and decided to remove it instead :)